src/frontend/staff-portal/src/app/components/jj-workbench/jj-dispute-digital-case-file/jj-dispute-digital-case-file.component.ts

```diff
@@ -62,12 +70,16 @@ export class JJDisputeDigitalCaseFileComponent implements OnInit, AfterViewInit
     this.dataSource.sort = this.sort;
   }
 
-  backWorkbench(element) {
+  backWorkbench(element: JJDispute) {
```

Comment:
I think we need to change the implementation, this is POC stuff... just saying not your fault tho